ts/data/case.ts

```diff
@@ -172,9 +172,9 @@ export const caseDefinition: TableDefinition<Case, Test> = {
         fragment.appendChild(nameDiv);
 
         const externalNameDiv = document.createElement("div");
-        externalNameDiv.appendChild(
-          document.createTextNode(kase.donor.externalName)
-        );
+        externalNameDiv.appendChild(document.createTextNode(kase.donor.externalName));
+        const tooltipInstance = Tooltip.getInstance();
+        tooltipInstance.addTarget(externalNameDiv, makeNameDiv("External Name", ""));
```

Comment:
Can use `document.createTextNode` instead of `makeNameDiv` since it doesn't need all of the links. Same in the other instances

Reply:
Done